src/bug-filing/azure-boards/create-azure-boards-issue-filing-url.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as _ from 'lodash/index';
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { HTTPQueryBuilder } from '../common/http-query-builder';
+import { IssueFilingUrlStringUtils, IssueUrlCreationUtils } from '../common/issue-filing-url-string-utils';
+import { buildTitle } from '../github/create-github-bug-filing-url';
+import { AzureBoardsBugFilingSettings } from './azure-boards-bug-filing-service';
+
+const buildTags = (createBugData: CreateIssueDetailsTextData, standardTags: string[]): string => {
+    const tags = ['Accessibility', title, `rule:${createBugData.ruleResult.ruleId}`, ...standardTags];
+    return tags.join('; ');
+};
+
+const buildBodySection = (headingHtml: string, contentHtml: string): string => {
+    return `<div><b>${headingHtml}</b></div><div>${contentHtml}</div><br>`;
+};
+
+const buildIssueDetails = (help: string, helpUrl: string, ruleId: string): string => {
+    const helpEscapedForHtml = _.escape(help);
+    const ruleIdEscapedForHtml = _.escape(ruleId);
+
+    return `${helpEscapedForHtml}<br><a href="${helpUrl}">${ruleIdEscapedForHtml}</a>`;
+};
+
+const buildSnippetHtml = (snippet: string): string => {
+    const maxSnippetLength = 256;
+    let constrainedSnippet = snippet;
+    if (snippet.length > maxSnippetLength) {
+        constrainedSnippet = snippet.substr(0, maxSnippetLength) + '...';
+    }
+    return _.escape(constrainedSnippet);
+};
+
+const buildApplicationHtml = (pageTitle: string, pageUrl: string): string => {
+    const pageTitleEscapedForHtml = _.escape(pageTitle);
+    const pageUrlEscapedForUrl = encodeURI(pageUrl);
+    const pageUrlEscapedForHtml = _.escape(pageUrl);
+
+    return `<a href="${pageUrlEscapedForUrl}">${pageTitleEscapedForHtml}<br>${pageUrlEscapedForHtml}</a>`;
+};
+
+const buildHowToFixHtml = (failureSummary: string): string => {
+    return _.escape(failureSummary)
+        .replace(/\n  /g, '<br>- ')
+        .replace(/\n /g, '<br> ')
+        .replace(/\n/g, '<br>');
+};
+
+const buildBody = (
+    createBugData: CreateIssueDetailsTextData,
```

Comment:
Just did. Some thoughts about this:
- our html output doesn't have much interactive elements (expect for the links)
- not much html is used expect for `<div>`'s and `<br>`'s
- if we find issues, we may be not be able to resolve them since Azure Boards is the one rendering our content.
I'll check with an AT before resolving this comment.

Reply:
I think we have some issues with our links but I would like to show the repro steps content to Fernanda, finalize the details and then fix any issue we may have. Sounds good @haonliu ?

Reply:
Sounds good!